### PR TITLE
Successful working keybinds in the games

### LINKED ISSUE #167: Keyboard control customization
For accessibility and customizability there should be a settings submenu for changing all keyboard (and mouse) bindings that are used by the game. Though not an urgent feature, it will help players using different keyboard layout. Especially if the text chat is a critical game component, using a different layout than usual would be very uncomfortable for some players. 

Support for multiple keybinds on one action would be a feature to consider (e.g. allowing movement with WASD and arrow keys). 

The implementation might be linked to and possibly dependent of #99.

### PR DESCRIPTION
**Pull request by zapzoop0099**
I have Successfully created the keybinds in the games for now this is for movement only but can be added for other stuff easily please review it and the option is available in settings menu

### CODE REVIEW

**Review by nicemicro — DISMISSED** [review]
It seems good to me, tested and it works fine.
If anyone else wants to review code quality, be my guest, I just scrolled through as far as I'm concerned it looks good.
EDIT: based on @vainiovano 's comment, I revoked my review.

**Comment by vainiovano** [comment]
The keybind system shouldn't be that separated from the other settings. In this implementation, editing keybinds modifies the game's source directory. The keybinds should be saved with other settings in `user://settings.cfg`. If using the same settings system isn't possible, the keybinds should still be in the user directory to preserve the source code directory.

**Comment by zapzoop0099** [comment]
The code is now perfect and using the same settings.cfg file of game and also saving in user home directory

File: src/assets/autoload/uimanager.gd [hunk 1]
```diff
@@ -21,7 +25,18 @@ var interactUINode: Node
 signal open_menu
 
 func _ready():
-# warning-ignore:return_value_discarded
+	write_keybinds()
```
Comment (vainiovano) [inline src/assets/autoload/uimanager.gd:28]: The keybinds shouldn't be reset each time the game is started.

File: src/assets/ui/submenus/settings/Keybind/KeyBind.gd [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		button.set_h_size_flags(Control.SIZE_EXPAND_FILL)
+		
+		if key == "ui_up":
+			label.text = "UP"
+		elif key == "ui_down":
+			label.text = "DOWN"
+		elif key == "ui_left":
+			label.text = "Left"
+		elif key == "ui_right":
+			label.text = "Right"
```
Comment (vainiovano) [inline src/assets/ui/submenus/settings/Keybind/KeyBind.gd:27]: inconsistent case

**Review by vainiovano — CHANGES_REQUESTED** [review]

**Comment by zapzoop0099** [comment]
I have closed this pull request And will write a new one as commit to these pr are now useless

**Pull request closed without merging**